Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1

```diff
@@ -296,11 +307,22 @@ function Disable-RemoteDesktopAccess
     {
         if ($totalCloudServiceWithEnableRDPAccess -gt 0)
         {
-            # Creating the log file
-            Write-Host "Backing up config of cloud service(s) detail." -ForegroundColor $([Constants]::MessageType.Info)
-            $cloudServiceWithEnabledRDPAccess | ConvertTo-json | out-file "$($folderpath)\CloudServiceWithEnabledRDPAccess.json"  
-            Write-Host "Path: $($folderpath)\CloudServiceWithEnabledRDPAccess.json"     
-            
+            if ($DryRun)
+            {
+                Write-Host "Exporting configurations of cloud service(s) having Remote Desktop enabled. You may want to use this CSV as a pre-check before actual remediation." -ForegroundColor Cyan
```

Comment:
We should say, 'Exporting list of  cloud service(s) having Remote Desktop enabled' bcz I dont think in CSV we put any extra information.